lib/set-path.js

```diff
@@ -1,6 +1,6 @@
 const {resolve, dirname} = require('path')
 const isWindows = require('./is-windows.js')
-const nodeGypPath = resolve(__dirname, 'node-gyp-bin')
+const nodeGypPath = resolve(__dirname, '../lib/node-gyp-bin')
```

Comment:
? shouldn't these be the same, since `set-path.js` is already in the `lib` folder?

Reply:
they are, this is changed to make the posix tests pass in windows. i realize that's a strange statement to make, but the tests for set-path were already written in such a way that we test both platforms regardless of where we're running.

the issue this resolves is that when using `require('path').posix` the call to `path.resolve(__dirname, 'node-gyp-bin')` returns something like `C:\Users\Nathan LaFreniere\Projects\run-script/C:\Users\Nathan LaFreniere\Projects\run-script\lib/node-gyp-bin` while making the second parameter a relative path returns the expected result.

i decided rather than turning off tests that we were already running to just go ahead and fix them